crates/krun-guest/src/cli_options.rs

```diff
@@ -26,7 +26,7 @@ pub fn options() -> OptionParser<Options> {
             .context("Failed to parse GID")
             .map(|gid| gid.into())
     });
-    let command = positional("COMMAND");
+    let command = positional("COMMAND").optional();
```

Comment:
Actually I don't think this can work correctly. What does an optional positional argument that's followed by non-optional one(s) even mean?

Reply:
That's a left over, I'll fix it.

Reply:
Done.